src/js/structs/StateSummary.js

```diff
@@ -53,7 +61,9 @@ module.exports = class StateSummary {
   }
 
   getActiveSlaves() {
-    return _.where(this.snapshot.slaves, {active: true});
+    return this.snapshot.slaves.filter(function (slave) {
```

Comment:
there is a slight difference here, where previously we would only get things that had `active === true`, now we get things that are truthy and not exactly true. probably won't matter, though